docs/flutterinfinitelisttutorial.md

```diff
@@ -20,15 +19,50 @@
 
 We’ll start off by creating a brand new Flutter project
 
-[script](_snippets/flutter_infinite_list_tutorial/flutter_create.sh.md ':include')
+```sh
+flutter create flutter_infinite_list
+```
 
 We can then go ahead and replace the contents of pubspec.yaml with
 
-[pubspec.yaml](_snippets/flutter_infinite_list_tutorial/pubspec.yaml.md ':include')
+[pubspec.yaml](https://raw.githubusercontent.com/felangel/bloc/master/examples/flutter_infinite_list/pubspec.yaml ':include')
```

Comment:
awesome! Can we delete the old snippet for the pubspec.yaml?

Reply:
Yes we can, and we can do this update for all the relevant snippets in the tutorials

Reply:
Removed all the old snippets from this tutorial